ambari-server/src/main/java/org/apache/ambari/server/audit/event/request/ClusterPrivilegeChangeRequestAuditEvent.java

```diff
@@ -99,7 +97,7 @@ protected void buildAuditMessage(StringBuilder builder) {
         }
       }
 
-      builder.append(StringUtils.join(lines, System.lineSeparator()));
+      builder.append(StringUtils.join(lines, " , "));
```

Comment:
the separator should be `;` since `,` is used to separate the values for each entry.